cibuildwheel/windows.py

```diff
@@ -77,10 +108,21 @@ def shell(args, env=None, cwd=None):
     temp_dir = tempfile.mkdtemp(prefix='cibuildwheel')
     built_wheel_dir = os.path.join(temp_dir, 'built_wheel')
 
-    python_configurations = get_python_configurations(build_selector)
+    def call(args, env=None, cwd=None, shell=False):
+        # print the command executing for the logs
+        if shell:
+            print('+ %s' % args)
+        else:
+            print('+ ' + ' '.join(shlex_quote(a) for a in args))
+
+        return subprocess.check_call(args, env=env, cwd=cwd, shell=shell)
```

Comment:
No. My mistake. Why using different names in macos and windows file?

Reply:
Err... I think because the `call(...)` functions run programs directly unless `shell=True` (similar to subprocess.call), but the `shell(...)` function always runs via a shell, because they're called using `cmd /C`